specification/components/schemas/Communication.yaml

```diff
@@ -1,5 +1,5 @@
 type: object
-description: A language that may be used to communicate with the patient.
+description: A language that may be used to communicate with the patient. This will only be populated on a retrieval and not a search.
```

Comment:
Raise a ticket for this to be TA reviewed

Reply:
https://jira.digital.nhs.uk/browse/APM-474 raised